src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,10 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        StringBuilder clean = new StringBuilder(text.replaceAll("[^a-zA-Z0-9]", "").toLowerCase());
+        String cleanStr = clean.toString();
+        String cleanReversed = clean.reverse().toString();
```

Comment:
```suggestion
        String textReversed = clean.reverse().toString();
```

Reply:
Here if I use String cleanText = new StringBuilder(...) ; then I might not have a possibility to use cleanText.reverse(); , I would need to create StringBuilder, what I did in the beggining of my code. Right?

Reply:
ok, but rename the variable